docker-compose.yaml

```diff
@@ -1,98 +0,0 @@
-version: '3'
```

Comment:
I still very doubt this approach with extracting docker deps into a separate repo. Yes we do decrease duplication this way, but compare please:

1. A curious developer starts to work with a NodeJS Backend:
    - git clone
    - docker-compose up -d
And it works! Two actions and it works! Now he/she can bring the changes on. Yes, we have to do more work supporting these files but it's our job to do. 
2. I haven't count steps for the kv-python version, there are much more actions.

#EasyAndObvious

Reply:
Actually, I'm not even sure if I can run the KV with a python backend using the instructions given. kv-python is meaningless without C*, Web and others, and the instructions lead me to another git repo there are bunch of other instructions not helping me to run the infra with the python.